clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -1663,6 +1663,13 @@ void clang::InitializePreprocessor(Preprocessor &PP,
     AddImplicitInclude(Builder, Path);
   }
 
+  // Process -include-internal-header directive.
```

Comment:
Is this comment accurate? Is  the 'processing' happening inside the function call?

Reply:
Yes. A `#include header.h` is added to the stream.